pom.xml

```diff
@@ -345,22 +346,18 @@
                                 </goals>
                                 <phase>test</phase>
                             </execution>
-                            <execution>
-                                <id>build-native</id>
-                                <goals>
-                                    <goal>build</goal>
-                                </goals>
-                                <phase>package</phase>
-                            </execution>
                         </executions>
                         <configuration>
-                            <imageName>sqlite-jdbc-osinfo</imageName>
-                            <mainClass>org.sqlite.util.OSInfo</mainClass>
+                            <fallback>false</fallback>
```

Comment:
what does this do exactly?

Reply:
A fallback image is basically a wrapper around the jars that executes the jars. It still requires a JDK to run and it isn't a truly native executable. It gets created when something in the configuration is insufficient and the native-image generation would otherwise crash.
Its sole purpose is to still create something that works for any java program you throw at it, which is as I understand is purely a legal requirement to call native-image a valid java implementation. For the same reason it is enabled by default but when you're after a truly native executable it's something you never want. So sadly it's something that always has to be disabled manually. A bit more info here, but as evident by all the legal disclaimers it's not easy to get a straight answer on the matter: https://github.com/oracle/graal/issues/2648. I am also not a lawyer so use that info with this in mind.